iocore/dns/DNSConnection.cc

```diff
@@ -78,6 +78,12 @@ void
 DNSConnection::trigger()
 {
   handler->triggered.enqueue(this);
+
+  // The handler should be pionting to DNSHandler::mainEvent.
```

Comment:
That's obvious. The question, is, what is accomplished by calling the main handler? Why does it need to be called from here, when it didn't before?